ramls/schemas/userdataimport.json

```diff
@@ -34,6 +34,14 @@
         "type": "string"
       }
     },
+    "departments": {
+      "description": "Names of departments the user belongs to",
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "type": "string"
+      }
+    },
```

Comment:
This adds support for "departments" property that mod-users v17.2.0 has added: https://github.com/folio-org/mod-users/commit/ffe061103c02b123cc9a6e219da30104cb690429
It skips the "preferredFirstName" property that mod-users v17.1.0 has added: https://github.com/folio-org/mod-users/commit/16b4e9845d3a349261ddd98c31d92631da794cf8
Can you add "preferredFirstName" as part of this pull request (and reword the Jira), or can you create a new Jira for that?

Reply:
Done